coinlib/bin/util.dart

```diff
@@ -29,7 +29,11 @@ Future<int> execWithStdio(
     await process.stdin.close();
   }
 
-  await process.stdout.transform(utf8.decoder).forEach(stdout.write);
+  // Pipe stdout to terminal and discard stderr
+  await Future.wait([
+    process.stdout.transform(utf8.decoder).forEach(stdout.write),
+    process.stderr.drain<String?>(),
```

Comment:
Does it work if you process both stdout and stderr the same way like this?

```dart
  await Future.wait([
    for (final stream in [process.stdout, process.stderr])
      stream.transform(utf8.decoder).forEach(stdout.write),
  ]);
```

Reply:
That works too, and the stderr (aka. the Docker build log) gets printed as expected  
Although, the print is huge ~1154 lines, so I'm not sure if it's better to print or drop stderr

Reply:
What does Docker output to stderr? If it works nicely, it shouldn't print anything to stderr.

Reply:
OK thank you. I think it's best to keep both stdout and stderr. I'll merge and then make the change myself.